templates/controllers/wizard/fileUpload/form/fileUploadForm.tpl

```diff
@@ -193,9 +193,6 @@
 			{include file="controllers/fileUploadContainer.tpl" id="plupload"}
 		{/fbvFormSection}
 
-		{translate|assign:"maxFileUploadStatus" key="common.fileUpload.maxFileSizeStatus" fileSize=$maxFileUploadSize supportName=$currentContext->getSetting('supportName') supportEmail=$currentContext->getSetting('supportEmail')}
```

Comment:
Should the variable maxFileUplaodSize then also be removed from the Form (class)?